torchgeo/losses/qr_losses.py

```diff
@@ -0,0 +1,53 @@
+"""Loss functions for learing on the prior."""
+
+from typing import cast
+
+import torch
+import torch.nn.functional as F
+
+
+def loss_on_prior_simple(log_probs: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
```

Comment:
PyTorch loss functions are defined as nn.Modules. Could we do that here for more consistency?